psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1510,22 +1557,25 @@ std::vector<SharedMatrix> MintsHelper::ao_nabla() {
     return nabla;
 }
 
-std::shared_ptr<CdSalcList> MintsHelper::cdsalcs(int needed_irreps, bool project_out_translations,
-                                                 bool project_out_rotations) {
-    return std::make_shared<CdSalcList>(molecule_, factory_, needed_irreps, project_out_translations,
-                                        project_out_rotations);
+
+std::shared_ptr <CdSalcList> MintsHelper::cdsalcs(int needed_irreps,
+                                                  bool project_out_translations,
+                                                  bool project_out_rotations)
+{
+    return std::shared_ptr<CdSalcList>(new CdSalcList(molecule_, factory_,
+                                                      needed_irreps,
+                                                      project_out_translations,
+                                                      project_out_rotations));
 }
 
-SharedMatrix MintsHelper::mo_transform(SharedMatrix Iso, SharedMatrix C1, SharedMatrix C2, SharedMatrix C3,
-                                       SharedMatrix C4) {
+SharedMatrix MintsHelper::mo_transform(SharedMatrix Iso, SharedMatrix C1, SharedMatrix C2,
+                                       SharedMatrix C3, SharedMatrix C4)
+{
     // Attempts to transform integrals in the most efficient manner. Will transpose left, right
     // and left_right where left and right are (left|right) indices. Does not consider the fimal
     // perturbation eg (12|34) -> (13|24), therefore integrals of type (oo|vv) will not be computed
     // in the optimal order. However, the first transformed index is guaranteed to be the smallest.
 
-    if ((C1->nirrep() + C2->nirrep() + C3->nirrep() + C4->nirrep()) > 4) {
-        throw PSIEXCEPTION("MO Transform: Incoming orbitals must be C1 symmetry.");
-    }
```

Comment:
Any reason to remove this? I still think its needed?

Reply:
Most of these changes are related to me not resolving the merge conflicts properly while rebasing.